Cannot find module './lib/commonjs/plugin/withUUIDplugin'

<!---
Thank you for taking the time to report bugs!

We love code snippets and links to repositories that reproduce the issue, but understand if you don't have the time to add them. We'll do our best with the info you provide, and might ask follow-up questions.

Please see our [OSS process document](https://github.com/honeycombio/home/blob/main/honeycomb-oss-lifecycle-and-practices.md#) to get an idea of how we operate.
--->

## Versions
- @honeycombio/opentelemetry-react-native 0.7.2
- Expo SDK 53 (expo ~53.0.20)
- React Native 0.79.5
- Android build via EAS (Linux, case-sensitive filesystem)
- macOS 26.0.1 (case-insensitive, iOS builds succeed)
## Steps to reproduce
- Install @honeycombio/opentelemetry-react-native@0.7.2 in an Expo project
- Run eas build --platform android (or any Android build on a case-sensitive filesystem).
- Build fails while evaluating app.plugin.js.

## Additional context

```
Build failed

Cannot find module './lib/commonjs/plugin/withUUIDplugin'
Require stack:
- /home/expo/workingdir/build/node_modules/@honeycombio/opentelemetry-react-native/app.plugin.js
- /usr/local/eas-build-worker/node_modules/@expo/config/node_modules/@expo/config-plugins/build/utils/plugin-resolver.js
- /usr/local/eas-build-worker/node_modules/@expo/config/node_modules/@expo/config-plugins/build/plugins/withStaticPlugin.js
- /usr/local/eas-build-worker/node_modules/@expo/config/node_modules/@expo/config-plugins/build/plugins/withPlugins.js
- /usr/local/eas-build-worker/node_modules/@expo/config/node_modules/@expo/config-plugins/build/index.js
- /usr/local/eas-build-worker/node_modules/@expo/config/build/plugins/withConfigPlugins.js
- /usr/local/eas-build-worker/node_modules/@expo/config/build/Config.js
- /usr/local/eas-build-worker/node_modules/@expo/config/build/index.js
- /usr/local/eas-build-worker/node_modules/@expo/build-tools/dist/utils/appConfig.js
- /usr/local/eas-build-worker/node_modules/@expo/build-tools/dist/context.js
- /usr/local/eas-build-worker/node_modules/@expo/build-tools/dist/builders/android.js
- /usr/local/eas-build-worker/node_modules/@expo/build-tools/dist/builders/index.js
- /usr/local/eas-build-worker/node_modules/@expo/build-tools/dist/index.js
- /usr/local/eas-build-worker/src/services/worker/dist/service.js
- /usr/local/eas-build-worker/src/services/worker/dist/ws.js
- /usr/local/eas-build-worker/src/services/worker/dist/main.js
```

- The package publishes lib/commonjs/plugin/withUUIDPlugin.js (uppercase “P”)
- app.plugin.js requires withUUIDplugin (lowercase “p”)
- Case-insensitive filesystems (macOS/iOS) mask this
- Linux-based Android builders fail. 
- Workaround: patch app.plugin.js to require ./lib/commonjs/plugin/withUUIDPlugin
- I’m currently fixing this via patch-package.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## v.Next
 
+- fix: Fix case of withUUIDPlugin import to match file
+
 ## v0.7.2
 
 - fix: Remove `SourceMapUuid.txt` from UUID plugin, because it can be misleading

--- a/app.plugin.js
+++ b/app.plugin.js
@@ -1,1 +1,1 @@
-module.exports = require('./lib/commonjs/plugin/withUUIDplugin');
+module.exports = require('./lib/commonjs/plugin/withUUIDPlugin');

